2017-2018/botCode/pacbotSimulatorModule.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import os
+import robomodules as rm
+from messages import MsgType, message_buffers, PacmanState
+
+ADDRESS = os.environ.get("LOCAL_ADDRESS","localhost")
+PORT = os.environ.get("LOCAL_PORT", 11295)
+
+
+class PacbotSimulatorModule(rm.ProtoModule):
+    def __init__(self, addr, port):
+        self.subscriptions = [MsgType.PACMAN_COMMAND]
+        super().__init__(addr, port, message_buffers, MsgType, self.subscriptions)  # Frequency is 0 by default
+        self.current_location = (14, 7)  # I can't figure out how to import the starting location of pacbot
```

Comment:
Import from where?

Reply:
/why isn't it working?

Reply:
It's working. I wanted to import Pacman's starting location from variables.py in the gameEngine package instead of hardcoding the value as I did here but I couldn't figure out how

Reply:
You won't be able to import it from the variables.py file that you modified on this branch, as they are in very different directories and I suspect that file is not in the path when running the simulator. You could create a new file in the botcode folder though.

Reply:
That's actually a good idea. Make a new variables file in the botcode folder and store that in there.

Reply:
👍